node/src/protocol.rs

```diff
@@ -79,6 +79,17 @@ impl Payload for Message {
         }
     }
 
+    fn is_low_priority(&self) -> bool {
+        match self {
+            Message::Consensus(_) => false,
+            Message::DeployGossiper(_) => false,
+            Message::AddressGossiper(_) => false,
+            Message::GetRequest { .. } => true,
+            Message::GetResponse { .. } => false,
+            Message::FinalitySignature(_) => false,
```

Comment:
As I mentioned earlier, I think these values should be configurable, so we can change them between test runs. This can be addressed in a separate PR, if at all.